eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -66,23 +66,29 @@ jobs:
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'Linux_musl_x64') }}:
       - ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        - (Alpine.312.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200602002622-e06dc59
+        - (Alpine.313.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-helix-amd64-20210910135845-8a6f4f3
```

Comment:
Now that we confirmed it worked for both, can you please comment one (probably 3.14)? CoreCLR testing is quite expensive.

Reply:
Given it runs in outerloop tests in parallel on Helix (when PRs are merged and no one is 'waiting'), do you think 3.13-only is good enough? Empirically speaking, these legs typically complete much faster (<= glibc distros' legs). Most time is spent on simulator and mobile platform jobs (which offsets everything else in terms of timing).

Reply:
I defer to @jkotas, he's investigated the impact of our testing on resources more heavily lately. My main qualm is the Helix queue this is running on, the ubuntu.1604.amd64 variants, has 200 machines shared for every dotnet repo and it's got pretty high demand. Even if running fast, they can only take so much work in parallel and everyone wants them. The 5.000.000+ tests per CI are already stressing that - add more to that per run and it will creep on us pretty badly; until we have better retries and a cleaner outerloop run, adding to outerloop is just adding to our pressure and to a long list of "yeah, things are broken. File an issue,"  so I've been generally skeptic on the value we get vs how expensive it is.

Reply:
Commented out 3.13 helix legs.